skimage/graph/tests/test_pixel_graph.py

```diff
@@ -46,3 +46,15 @@ def test_default_edge_func():
     assert num_edges == 12  # lattice in a (3, 3) grid
     np.testing.assert_almost_equal(g[0, 1], 0.78 * np.abs(image[0, 0] - image[0, 1]))
     np.testing.assert_array_equal(n, np.arange(image.size))
+
+
+def test_no_mask_with_edge_func():
+    """Ensure function `pixel_graph` runs when passing `edge_function` but not `mask`."""
+    image = np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]])
+
+    def func(x, y, z):
+        return np.abs(x - y)
+
+    g = pixel_graph(image, edge_function=func)
+    assert g.shape == (9, 9)
+    assert len(g.data) == 24
```

Comment:
Sorry, the return value has length 2 (feel free to double-check); it should be:
```suggestion
    assert g[0].shape == (9, 9)
    assert len(g[0].data) == 24
    np.testing.assert_array_equal(g[1], np.array(range(9)))
```

Reply:
Can we maybe write in the same format as other tests? i.e. 

```
    g, n = pixel_graph(image, edge_function=func)
    assert g.shape == (9, 9)
    assert len(g.data) == 24
    np.testing.assert_array_equal(n, np.arange(image.size))
```

Reply:
Absolutely!